SimPEG/Examples/MT_1D_ForwardAndInversion.py

```diff
@@ -100,7 +100,7 @@ def run(plotIt=True):
     # Regularization - with a regularization mesh
     regMesh = simpeg.Mesh.TensorMesh([m1d.hx[problem.mapping.sigmaMap.maps[-1].indActive]],m1d.x0)
     reg = simpeg.Regularization.Tikhonov(regMesh)
-    reg.smoothModel = True
+    reg.mrefInSmooth = True
```

Comment:
FYI `smoothModel = True` is equivalent to `mrefInSmooth = False` (so the nature of this example changed). Not sure if that was intentional?